synapse/config/saml2_config.py

```diff
@@ -228,6 +226,11 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
         #
         # You will also want to investigate the settings under the "sso" configuration
         # section below.
+
+        # Enable SAML2 for registration and login. Uses pysaml2.
+        #
+        # At least one of `sp_config` or `config_path` must be set in this section to
+        # enable SAML login.
```

Comment:
I moved this section down since the bits above apply to all of the SSO code. I actually wonder if it should be part of the `sso_config` class and that should go before SAML/OIDC/CAS? 🤷